{
	"name": "CVE-2017-16228",
	"threat_severity": "Moderate",
	"public_date": "2017-10-29T00:00:00Z",
	"bugzilla": {
		"description": "python-dulwich: Setting SSH arguments from untrusted URLs allows code execution",
		"id": "1509303",
		"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1509303"
	},
	"cvss3": {
		"cvss3_base_score": "7.8",
		"cvss3_scoring_vector": "CVSS:3.0/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
		"status": "draft"
	},
	"cwe": "CWE-20",
	"details": [
		"Dulwich before 0.18.5, when an SSH subprocess is used, allows remote attackers to execute arbitrary commands via an ssh URL with an initial dash character in the hostname, a related issue to CVE-2017-9800, CVE-2017-12836, CVE-2017-12976, CVE-2017-1000116, and CVE-2017-1000117."
	],
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2017-16228\nhttps://nvd.nist.gov/vuln/detail/CVE-2017-16228"
	],
	"package_state": [
		{
			"cpe": "cpe:/a:redhat:openstack:11",
			"fix_state": "Will not fix",
			"package_name": "python-dulwich",
			"product_name": "Red Hat OpenStack Platform 11 (Ocata)"
		}
	],
	"upstream_fix": "python-dulwich 0.18.5",
	"csaw": false
}
